fragmentstest/src/main/java/org/odk/collect/fragmentstest/DialogFragmentTest.kt

```diff
@@ -15,25 +13,6 @@ import org.hamcrest.Matcher
  */
 object DialogFragmentTest {
 
-    /**
-     * Creates a FragmentScenario using a test theme that supports AppCompat and MaterialComponents
-     * attributes but also won't explode (like MaterialComponents themes do).
-     *
-     * This uses [FragmentScenario.launch] here instead of [FragmentScenario.launchInContainer] as
-     * per docs at https://developer.android.com/guide/fragments/test#dialog.
-     */
-    internal fun <F : Fragment> launchDialogFragment(
```

Comment:
I'd prefer to keep this where it is, so we have the `DialogFragment` in one place. Hopefully we can get rid of this method once the bugs in Robolectric/AndroidX are fixed.

Reply:
ok